src/components/attendee/AttendeeLayout.module.scss

```diff
@@ -51,29 +34,20 @@
   z-index: 40;
   @include sparkle.panel--normal;
 
-  @media (max-width: breakpoint.$md - 1) {
-    display: flex;
-    flex-direction: column;
+  display: flex;
+  flex-direction: column;
 
-    &[open] {
-      height: 45vh;
+  &[open] {
+    height: 45vh;
 
-      &.has-component-shoutout {
-        height: calc(45vh - #{space.button(1) + space.empty(1)});
-      }
+    &.has-component-shoutout {
+      height: calc(45vh - #{space.button(1) + space.empty(1)});
```

Comment:
I'm going to come back to this. It's messy.

Reply:
`@debt` comment?

Reply:
This isn't used. Removed it.